packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/tests/index.test.js.orig

```diff
@@ -0,0 +1,655 @@
+import React from 'react';
```

Comment:
this file has git stuff in it and a weird extension – is this a mistake?

Reply:
Oops, they must have creeped in with a bad merge. Thanks. Fixed in 8d175b1e69a567d1fb16515680b0482f60b8da60